lightkurve/correctors/tests/test_regressioncorrector.py

```diff
@@ -57,28 +57,29 @@ def test_sinusoid_noise():
     true_lc = LightCurve(time, true_flux, flux_err=0.1*np.ones(size))
     # Noisy light curve has a sinusoid single added
     noisy_lc = LightCurve(time, true_flux+noise, flux_err=true_lc.flux_err)
-    dm = DesignMatrix({'noise': noise,
+    design_matrix = DesignMatrix({'noise': noise,
                        'offset': np.ones(len(time))},
                       name='noise_model')
```

Comment:
:point_up:  It's just a test, but let's align the indents with the opening delimiters here -- makes the code :100:x more readable to me.